src/sentry/static/sentry/app/components/events/interfaces/frame/context.tsx

```diff
@@ -74,24 +79,7 @@ const Context = ({
           const isActive = frame.lineNo === line[0];
           const hasComponents = isActive && components.length > 0;
           return (
-            <ContextLine
-              key={index}
-              line={line}
-              isActive={isActive}
-              css={
-                hasComponents
```

Comment:
We can no longer conditionally apply the CSS like this since we won't know if the `StacktraceLink` should render. But it turns out we didn't need this (if no child components under `ContextLine` are there the component doesn't appear).